### ISSUE TITLE
A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_Keyboard : The "X" (Close) control is not actionable using keyboard and announced as text for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11542]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#Keyboard;#A11ySev2;#Benchmark;#MAS2.1.1;#MAS1.3.1;#MAS4.1.2;#FTP;

### Environment Details:
Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS2.1.1, MAS1.3.1, MAS4.1.2

### Repro Steps:

1. Launch Azure Data Studio Insiders application.
2. If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3. Start screen reader, navigate to "Preview" control and activate it.
4. A dialog appears on the screen, the "X" (Close) control is not actionable using keyboard and for screen reader users it is announced as text.

### Actual:
When keyboard users navigate through the dialog content the "X" (Close) control is not actionable using keyboard. Similarly for screen reader users this control is announced as text as it is not provided with a proper name and role for the "X" (Close) control which is incorrect.

### Expected:
The "X" (Close) control should be actionable using keyboard so that keyboard users can access it. For screen reader users the "X" (Close) control should be provide with a proper name as "Close" and a proper role as "button" should be provided.

### User Impact:
If a control present on the screen is not actionable using keyboard and even the name and role of that control is not announced for screen reader users then the keyboard and screen reader users will not understand the purpose of that control and this can lead to confusion.

### Attachment link for Reference:
[11542_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_Keyboard The "X" (Close) control is not actionable using keyboard and announced as text for screen reader users.zip](https://github.com/microsoft/azuredatastudio/files/4985337/11542_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_Keyboard.The.X.Close.control.is.not.actionable.using.keyboard.and.announced.as.text.for.screen.reader.users.zip)

**Comment by alanrenmsft** [comment]
I have simplified the preview information to use tooltip to show the text provided by the old dialog.

**Comment by Raghu-addepalli** [comment]
Issue is fixed Hence closing the bug and added attachment.
Githubtags:  #Closed;
[Uploading 11542_Fixed.zip…]()
